app.js

```diff
@@ -12,14 +12,45 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
+hbs.registerPartials(path.join(__dirname, 'views/partials/'));
+
 // Register the location for handlebars partials here:
 
 // ...
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render(__dirname + '/views/index');
 });
 
+app.get('/beers', (req, res) => {
+    punkAPI.getBeers()
+    .then((beersFromApi) => {
+      res.render(__dirname + '/views/beers', {beersFromApi});
+    })
+
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom()
+  .then((getRandom) => {
+    res.render(__dirname + '/views/random-beers', {getRandom});
+  })
+
+  .catch(error => console.log(error));
+});
+
+app.get('/beers/:beerId', (req, res) =>{
+
+  let beerId = req.params.beerId;
+  punkAPI.getBeer(beerId)
+    .then((data)=> {
+      let myBeer = data[0]
+      console.log(data[0].image_url)
```

Comment:
remember to remove lines of code that were used only to test, since you are passing the whole data element in line 52, then these two lines (50 and 51) are not used/needed.